extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1845,6 +1838,7 @@ export class AmpA4A extends AMP.BaseElement {
       }
     );
 
+    const extensionIds = extensions.map((extension) => extension.extensionId);
```

Comment:
Why only `extensionIds` here?

Reply:
How do they usually use it?

Reply:
Looks like its only used one place, https://github.com/ampproject/amphtml/blob/511cc8ce45d69a33dc0c5109446b9beac0ebf50d/extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js#L1285

This will get cleaned up with no signing (which got approved for launch this week).